JASP-Common/datasetpackage.cpp

```diff
@@ -13,6 +13,7 @@ void DataSetPackage::reset()
 	analysesHTML = std::string();
 	analysesData = Json::arrayValue;
 	hasAnalyses = false;
+	warningMessage = std::string();
```

Comment:
just FYI, this isn't necessary; strings are always initialised as empty and don't need to be explicitly constructed.